default.nix

```diff
@@ -110,7 +110,7 @@ rec {
   inherit reflex-platform;
   inherit (reflex-platform) nixpkgs pinBuildInputs;
   path = reflex-platform.filterGit ./.;
-  command = ghcObelisk.obelisk-command;
+  obelisk = ghcObelisk;
   shell = pinBuildInputs "obelisk-shell" ([
     command
```

Comment:
This needs to be `ghcObelisk.obelisk-command` now

Reply:
?

I've fixed the README.

Reply:
> $ nix-build -A shell 
> error: undefined variable 'command' at /Users/alexandreesteves/repos/obelisk/default.nix:115:5